### PR TITLE
DEVPROD-628 Add option to disable default search attributes

### PR DESCRIPTION
**Pull request by lilydoar**
## What was changed
Added an option to disable creating the default search attributes

## Why?
Scenarios running against cloud server cells cannot create search attributes through the SDK. They must use the control plane instead.

### CODE REVIEW

File: loadgen/scenario.go [hunk 1]
```diff
@@ -132,6 +132,11 @@ type RunConfiguration struct {
 	// If the timeout is hit any pending executions will be cancelled and the scenario will exit
 	// with an error. The default is unlimited.
 	Timeout time.Duration
+	// Do not register the default search attributes used by scenarios. If the SAs are not registered
+	// by the run, they must be registerd by some other method. This is needed because cloud cells
+	// cannot use the SDK to register SAs, instead the SAs must be registered through the control plane.
+	// Default is false.
+	NoDefaultSearchAttributes bool
```
[the remark this answers is not in the document]
Comment (lilydoar) [reply loadgen/scenario.go]: Good call. Changed

File: loadgen/scenario.go [hunk 2]
```diff
@@ -132,6 +132,11 @@ type RunConfiguration struct {
 	// If the timeout is hit any pending executions will be cancelled and the scenario will exit
 	// with an error. The default is unlimited.
 	Timeout time.Duration
+	// Do not register the default search attributes used by scenarios. If the SAs are not registered
+	// by the run, they must be registerd by some other method. This is needed because cloud cells
```
Comment (dandavison) [inline loadgen/scenario.go:136]:
```suggestion
	// by the run, they must be registered by some other method. This is needed because cloud cells
```

**Review by dandavison — APPROVED** [review]
Nice, LGTM.

**Review by Sushisource — APPROVED** [review]
Thanks!

**Pull request merged**